packages/visualizations/addon/components/cell-renderers/dimension.js

```diff
@@ -10,14 +10,63 @@
  * }}
  */
 
-import Ember from 'ember';
+import Component from '@ember/component';
+import { computed, get } from '@ember/object'
 import layout from '../../templates/components/cell-renderers/dimension';
 
-export default Ember.Component.extend({
+export default Component.extend({
   layout,
 
   /**
    * @property {Array} classNames - list of component class names
    */
-  classNames: ['table-cell-content', 'dimension']
+  classNames: ['table-cell-content', 'dimension'],
+
+  /**
+   * @property {String} descField - field to find the description
+   */
+  descField: computed('column.field.dimension', function() {
+    return get(this, 'column.field.dimension') + '|desc';
+  }),
+
+  /**
+   * @property {String} idField - field to find id
+   */
+  idField: computed('column.field.dimension', function() {
+    return get(this, 'column.field.dimension') + '|id';
+  }),
+
+  /**
+   * @property {String} title - value that should be used in hoverover title
+   */
+  title: computed('descField', 'idField', 'data', function() {
```

Comment:
![screen shot 2018-08-10 at 10 23 55 am](https://user-images.githubusercontent.com/2703389/43966455-889e6fe2-9c87-11e8-83e3-c47b9660ab8d.png)

I wasn't sure what you meant by that. But I will explain what I meant.
In the table, unless you choose an column that displays the `id`, the `id` is not seen when a description is available. When we talked to some of the spotlight users, they were more familiar with the `id` than the `desc`. So on hover, if we can show both, that would be an improvement. `description (id)`.

Reply:
I was just saying desc is right next to the id in this case during hover over, but I changed this behavior. to be `desc (id)` if both are present.